src/main/java/org/jabref/gui/LibraryTabContainer.java

```diff
@@ -15,7 +15,7 @@ public interface LibraryTabContainer {
 
     void addTab(BibDatabaseContext bibDatabaseContext, boolean raisePanel);
 
-    void closeTab(LibraryTab libraryTab);
+    boolean closeTab(LibraryTab libraryTab);
```

Comment:
JavaDoc would be nice (if not much effort). Why does `closeCurrentTab()` still return void?